src/test/java/StringManipulationTest.java

```diff
@@ -118,32 +218,69 @@ public void testRestoreString1()
         assertEquals(restoreString, "rat");
     }
 
+    // This will test the method if it will swap the two words in the string.
```

Comment:
Does this array index work for both whitespace character since each len() index is considered a string character?

Reply:
Alvin, thanks for your comment! Yes, if I understand your comment correctly, this array will work for whitespace characters since whitespaces are considered strings, or substrings at the specific index.

Based on your comment, I've refactored my code to add an additional test that will explicitly test the whitespace characters. Please see my next push update for the additional test that specifically addresses this comment.